packages/itwin/map-layers/eslint.config.mjs

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import iTwinPlugin from "@itwin/eslint-plugin";
+import reactPlugin from "eslint-plugin-react";
```

Comment:
shouldnt we use the itwin ui eslint plugin config?
https://github.com/iTwin/eslint-plugin/blob/main/dist/configs/ui.js

Reply:
We do, on line 16